src/integrationTest/java/uk/gov/hmcts/reform/wataskmonitor/clients/camunda/CamundaClientTest.java

```diff
@@ -58,11 +57,11 @@ void givenCamundaClientIsCalledShouldReturnTasks() throws IOException {
                 null,
                 ZonedDateTime.parse(
                     "2021-06-02T16:17:32.808+0000",
-                    CAMUNDA_DATA_TIME_FORMATTER.withZone(ZoneId.of("UTC"))
+                    CAMUNDA_DATA_TIME_FORMATTER
```

Comment:
Does this default to UTC?

Reply:
This doesn't really defaults to UTC. But it is converting the date time string to ZonedDateTime Ravi.

Reply:
But a ZonedDateTime usually has a Zone, so which Zone is it going to use here?  Previously it was `UTC` but is it now just the Zone of the machine the test is running in?  I am just thinking of why you had to remove the Zone and if it's going to become inconsistent.

Reply:
We discussed and looks fine.